sdk/batch/Microsoft.Azure.Management.Batch/CHANGELOG.md

```diff
@@ -1,10 +1,19 @@
 # Release History
 
-## 10.1.0-preview.1 (Unreleased)
+## 11.0.0 (2020-06-01)
+### REST API version
+- This version targets REST API version 2020-05-01.
+
+### Features
+- Added `BatchAccountIdentity` property on `BatchAccount` for enabling system assigned identity when `Microsoft.KeyVault` is specified as the `BatchAccount` encryption property.
+
+### Bug Fixes
+- **[Breaking]** Convert the `PrivateEndpointConnection` update operation to a long running operation.
 
 ## 10.0.0 (2020-04-11)
```

Comment:
maybe put deprecated next to the version.

Reply:
It's technically not deprecated and won't be for at least a year after release. I think a warning that it is not recommended, and that there are known issues should be sufficient.